src/ts/component/block/text.tsx

```diff
@@ -859,7 +863,7 @@ const BlockText = observer(class BlockText extends React.Component<Props> {
 				focus.set(block.id, { from: range.from + l, to: range.to + l });
 				focus.apply();
 			});
-
+			{}
```

Comment:
Unused braces?

Reply:
I tested how vscode handles indentation on the same page, but I forgot to delete it, which was a mistake that should not have happened.